epistles-v2/original/06-epistles.adoc

```diff
@@ -420,6 +425,7 @@ So with my love in the everlasting seed.
 [.signed-section-signature]
 G+++.+++ F.
 
+[.signed-section-context-close]
 Swarthmore, the 26th of the 7th month, 1678.
 
 [.centered]
```

Comment:
Below here, on line `435` where it says `- To Friends in the County of 0xford`, can you make _two_ quick tweaks?

1) Remove the `-`
2) the `O` is actually a _zero_, can you make it an `O` instead?